simpledotorg/simple-server
# Translations for Screening & Supplies Reports

**Story card:** [sc-10309](https://app.shortcut.com/simpledotorg/story/10309/translations-for-screening-supplies-reports)

## Because

To translate Screening report for Ethiopia & Supplies report for all countries

## This addresses

Adds english texts for translating to other languages.

## Test instructions

1. For ET Screening form, cross-check text with this [finalized layout in Figma](https://www.figma.com/file/vmZlmlFSCOrEYzTMkqsNB8/Simple-(Production)?node-id=29876-50212&t=8kEMClDQ4fwxTGvJ-0)
2. For Supplies report, cross-check text with this [Exloration in Figma under section: `Supplies report (exploration for Sri Lanka/Ethiopia/Bangladesh)`](https://www.figma.com/file/kONxBLSXwcQDnV7ya4hIXA/Simple-(Exploration)?node-id=24690-179468&t=MpUmwovT3Zi7m9qX-0)
